src/components/ButtonGroup/ButtonGroup.stories.mdx

```diff
@@ -83,7 +83,7 @@ width.
 
 This component is a wrapper around the `Button` component. For accessibility
 information on the `Button` component, see the
-[Button Accessibility](https://nypl.github.io/nypl-design-system/reservoir/v0/?path=/docs/components-form-elements-button--button-with-controls#accessibility)
+[Button Accessibility](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/docs/components-form-elements-button--button-with-controls#accessibility)
```

Comment:
Is this a change that was missed in the PR for `DSD-958`?

Reply:
Or added since that PR.